Gaijinizer patch/Map166.txt

```diff
@@ -122,9 +130,9 @@ entrusted him with finishing the book.#Triketos
 我が主も、ご存知のはずでしょう？
 > CONTEXT: Map166/events/9/pages/0/60/Dialogue/other1
 > CONTEXT: Map195/events/9/pages/0/57/Dialogue/other1
-First of all...From the time when I was called
-"Kuroiso", the familiarism is not created
-principle...Should I also know my Lord?#MTLed
+Firstly, from the time I was known as the
+"dark fox", I have resolved never to create any
+familiars. You must surely know this my Lord.#Ron
 > END STRING
```

Comment:
Capitalize "Dark Fox" here. (This will become very important _**much**_ later...)